### ISSUE TITLE
Modal

### ISSUE THREAD

**Issue by halimahLDN** [Greater-London-Authority/ldn-viz-tools#750]
- [x] Insufficient colour contrast:
![image](https://github.com/user-attachments/assets/43a9d457-af97-4d91-a789-f7df74f32f3a)
![image](https://github.com/user-attachments/assets/6832c6ab-7d44-4bfb-ab55-3e5107637f4e)
![image](https://github.com/user-attachments/assets/0aa09abe-d80e-4e34-93ae-19b803a9b893)

- [x] Make it possible to scroll main content on keyboard
It's not possible to tab into the main content to scroll with an arrow key. When the modal opens the `Close` button is focused and pressing the down key does nothing. The next tabbed element is the `Accept` button, skipping the main content completely.

- [x] Screen reader doesn't read information in modal when it pops up
